.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,19 @@
+{
+	"name": "Immich devcontainers",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			"BASEIMAGE": "mcr.microsoft.com/devcontainers/typescript-node:22"
+		}
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"svelte.svelte-vscode"
+			]
```

Comment:
I think there's some more extensions worth adding, see https://immich.app/docs/developer/setup#vscode

Reply:
It's beyond my field of expertise. As I stated above, the purpose of my PR was to share my setup to ease contributions regarding web server (and doc).
I pretty sure that `vscode/settings.json` is still in use inside the dev container. Regarding missing extensions (maybe you are referring to flutter, ...), I've never used them so I won't be able to test anything.
If the change is easy, feel free to add a suggestion, otherwise, maybe open another PR ?